### PR TITLE
Restructure Fixes + Catalina for Chrome 88/89

### PR DESCRIPTION
**Pull request by blakebyrnes**

### CODE REVIEW

File: hero/examples/news.ycombinator.com.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import agent, { Observable } from 'secret-agent';
 // process.env.SA_SHOW_BROWSER = 'true';
 
 async function run() {
-  await agent.configure({ userAgent: '~ chrome >= 88' });
+  await agent.configure({ userAgent: '~ chrome = 88.*' });
```
Comment (calebjclark) [inline hero/examples/news.ycombinator.com.ts:6]: Why the asterisk?

File: lerna.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "hero/client",
     "hero/core",
     "hero/interfaces",
+    "here/plugin-utils",
```
Comment (calebjclark) [inline lerna.json:8]: I believe this should be "hero" not "here".

File: hero/examples/news.ycombinator.com.ts (hunk 1 above)
Comment (blakebyrnes) [reply hero/examples/news.ycombinator.com.ts:6]: compareVersions otherwise says nothing = 88.0.0

File: lerna.json (hunk 2 above)
Comment (blakebyrnes) [reply lerna.json:8]: Doh

File: hero/examples/news.ycombinator.com.ts (hunk 1 above)
Comment (blakebyrnes) [reply hero/examples/news.ycombinator.com.ts:6]: Of Chrome? No... The versions are like 88.0.4324.{many patches}

Comment (blakebyrnes) [reply hero/examples/news.ycombinator.com.ts]: It works without the asterisk, so I removed it from the example.

**Pull request merged**